docs/FiltersSummary.md

```diff
@@ -1,46 +1,49 @@
 # Filters
 
-By default, DotLiquid provides a set of [standard filters](https://github.com/Shopify/liquid/wiki/Liquid-for-Designers#standard-filters) to assist template creation.
+
+⚠ **This document applies to the Liquid engine. Follow [this](https://github.com/microsoft/FHIR-Converter/tree/handlebars) link for the documentation of Handlebars engine.**
+
+By default, Liquid provides a set of [standard filters](https://github.com/Shopify/liquid/wiki/Liquid-for-Designers#standard-filters) to assist template creation.
 Besides these filters, FHIR Converter also provides some other filters that are useful in conversion, which are listed below.
 If these filters do not meet your needs, you can also write your own filters.
 
-## Hl7 v2 specific filters
+### Hl7 v2 specific filters
 | Filter | Description | Syntax |
 |-|-|-|
-| get_first_segments | Returns first instance of the segments | `{% assign result = hl7v2Data | get_first_segments: 'segment1|segment2|...' -%}` |
-| get_segment_lists | Extracts HL7 v2 segments | `{% assign result = hl7v2Data | get_segment_lists: 'segment1|segment2|...' -%}` |
-| get_related_segment_list | Given a segment and related segment name, returns the collection of related named segments | `{% assign result = hl7v2Data | get_related_segment_list: parentSegment, 'childSegmentName' -%}` |
-| get_parent_segment | Given a child segment name and overall message index, returns the first matched parent segment | `{% assign result = hl7v2Data | get_parent_segment: 'childSegmentName', 3, 'parentSegmentName' -%}` |
-| has_segments | Checks if HL7 v2 message has segments | `{% assign result = hl7v2Data | has_segments: 'segment1|segment2|...' -%}` | 
+| get_first_segments | Returns first instance of the segments | `{% assign result = hl7v2Data \| get_first_segments: 'segment1\|segment2\|...' -%}` |
```

Comment:
Thanks very much for helping to fix the backslashes. It works fine in DevOps but has rendering errors on GitHub.